platform/windows/os_windows.cpp

```diff
@@ -1472,6 +1473,12 @@ Error OS_Windows::initialize(const VideoMode &p_desired, int p_video_driver, int
 			video_mode.fullscreen=false;
 		}*/
 		pre_fs_valid = false;
+
+		// If the user has mouse trails enabled in windows, then sometimes the cursor disappears in fullscreen mode.
+		// Save number of trails so we can restore when exiting, then turn off mouse trails
+		SystemParametersInfoA(SPI_GETMOUSETRAILS, 0, &restore_mouse_trails, 0);
+		if (restore_mouse_trails > 0)
+			SystemParametersInfoA(SPI_SETMOUSETRAILS, 0, 0, 0);
```

Comment:
You should use curly brackets after `if`, even with a single line statements. (Style requirement.)